examples/diff.c

```diff
@@ -47,6 +47,15 @@ enum {
 	CACHE_NONE = 2
 };
 
+#define DEFAULT_THRESHOLD 50
+#define DEFAULT_BREAK_REWRITE_THRESHOLD 60
+#define USE_DEFAULT(X) ((X) == 0 || (X) > 100)
+
+struct pair {
+	int first;
+	int second;
+};
```

Comment:
How to signal the caller that there has been error in input, by setting the values of first or second to negative ? Also will it be valid for the program to just die without explaining the cause of the error ?